src/test/java/picard/sam/SamErrorMetric/BaseErrorCalculationTest.java

```diff
@@ -13,6 +13,8 @@
 import java.io.IOException;
 import java.util.List;
 
+import static picard.cmdline.CommandLineProgramTest.CHR_M_REFERENCE;
```

Comment:
It would be nice to avoid static imports if possible, in this case you can refer to the constant using `CommandLineProgramTest.CHR_M_REFERENCE`